lib/protocol/alertAccept.js

```diff
@@ -26,7 +26,7 @@ let alertAccept = function () {
         method: 'POST'
     }
 
-    return this.requestHandler.create(requestOptions)
+    return this.requestHandler.create(requestOptions, {dummy: 'dummy'})
```

Comment:
When passing empty object to geckodriver directly, it returns error, that body data is invalid or missing. Looks like request library don't send data, if it's empty